src/static/js/modules/util/web-storage-proxy.js

```diff
@@ -11,90 +11,107 @@
    for local storage (note omitted 'true' value).
    Note: values stored in local storage are not accessible from session storage
    and vice versa. They both work on different objects within the browser.
-   ========================================================================== */
+   ========================================================================= */
 
 'use strict';
 
-// Default session type.
-// true = sessionStorage (non-persistent).
-// false = localStorage (persistent).
-var _sessionOnly = false;
+// Default storage type.
+var _storage = window.sessionStorage;
 
 /**
  * Set an item value specified by the key in web storage.
  * @param {string} key The key for the value.
  * @param {string} value The value to store.
- * @param {boolean} sessionOnly (Optional)
- *   Use non-persistent storage (true) or persistent storage (false).
+ * @param {object} storage (Optional)
+ *   Use non-persistent storage (sessionStorage)
+ *   or persistent storage (localStorage).
```

Comment:
From what I read, it's better to use `window.localStorage` because
> According to W3C standards, the correct is window.localStorage because the localStorage attribute is part of window object. And it is not a function, it is an attribute.

REF: http://stackoverflow.com/questions/5319878/which-is-the-better-option-to-use-localstorage-window-localstorage-or-doc

But I'll be honest, I still don't see the difference, just going by what I found when researching how best to build mocked storage.